packages/paste-website/src/components/site-wrapper/sidebar/SidebarAnchor.tsx

```diff
@@ -9,7 +9,9 @@ interface SidebarAnchorProps {
   onClick?: () => void;
 }
 
-const StyledSidebarAnchor = styled(Link)<SidebarAnchorProps>((props) =>
+const StyledSidebarAnchor = styled(Link, {
+  shouldForwardProp: (prop) => prop !== 'nested',
+})<SidebarAnchorProps>((props) =>
```

Comment:
Is this related to Codeblocks or a fix for something else?

Reply:
This is a fix for something else. I noticed that the console was spammed with a warning about a "nested" prop being passed to a DOM element. This fixes that.

Reply:
OMG I've been looking for this for months! 🙏🏼